src/software/backend/input/network/network_client.h

```diff
@@ -0,0 +1,166 @@
+#pragma once
```

Comment:
:exclamation: Copied from software/backend/input/network/networking with some refactoring and new functions